src/_pytest/reports.py

```diff
@@ -196,11 +196,17 @@ def head_line(self) -> Optional[str]:
             return domain
         return None
 
-    def _get_verbose_word(self, config: Config):
+    def _get_verbose_word(
+        self, config: Config, default_markup: Mapping[str, bool]
+    ) -> Tuple[str, Mapping[str, bool]]:
         _category, _short, verbose = config.hook.pytest_report_teststatus(
```

Comment:
*(non-blocking)* `verbose` here might benefit from being renamed to something more verbose. Like `word_and_maybe_markup`.

Reply:
I used _get_verbose_word_with_markup suggested in different comment since markup is always returned.